api4/user_test.go

```diff
@@ -1112,6 +1115,22 @@ func TestPatchUser(t *testing.T) {
 		t.Fatal("manualTimezone did not update properly")
 	}
 
+	err := th.App.CheckPasswordAndAllCriteria(ruser, *patch.Password, "")
+	if err == nil {
```

Comment:
FYI you can use the assert library here.

Reply:
For some particular reason nil checks doesn't work nicely together with AppError.